spec/requests/zips_request_spec.rb

```diff
@@ -40,7 +40,9 @@
     end
 
     it 'does a not found error' do
-      expect { get "/works/#{work_id}/zip" }.to raise_error ActionController::RoutingError
+      expect do
+        bypass_rescue { get "/works/#{work_id}/zip" }.to raise_error ActionController::RoutingError
```

Comment:
`bypass_rescue` is new to me. Is that a new Rails 7.2-ism we should get familiar with using? Or would reverting a change around here https://github.com/sul-dlss/happy-heron/pull/3635/files#diff-77f322f5ab0c54b1f2793b339574333dc55889645d910a93ede0fd7aa13b217aL42 get us out of needing to know about this?

Reply:
This test did not pass without this change. For me anyway, reverting.setting the config you note didn't change that.